### DISCUSSION TITLE
How to validate github workflow yaml syntax?

### DISCUSSION THREAD

**Discussion by bukowa** [community/community#36069] [Actions]
Hey i am really tired of pushing my github workflow just to get error after 10minutes of running because i missed `)`.
How can i make my life better and be less frustrated?

**Comment by sammcj** [comment]
It's far from perfect, and yes GitHub should have an official version but try Actionlint, it catches about 60% of things I find. https://github.com/rhysd/actionlint

**Reply by bukowa** [reply]
Thank you it looks cool!
